apps/02RockPaperScissors.js

```diff
@@ -9,15 +9,40 @@ var rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  hand1 = hand1.toLowerCase();
+  hand2 = hand2.toLowerCase();
 
   // Write code here
+  if (hand1 === hand2) {
+    return "It's a tie!";
+  } else if (hand1 === "rock") {
 
+    if (hand2 === "scissors") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
+    }
+  } else if (hand1 === "paper") {
+
+    if (hand2 === "rock") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
+    }
+  } else if (hand1 === "scissors") {
+
+    if (hand2 === "rock") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
+    }
+  }
```

Comment:
Try again - consider:

- What values are you expecting as input for the game? 
- When you run the game, what part of your code gets that input?
- In that part of your code, can you think of a way to check and see if the input matches what you're expecting?